src/classes/RD2_DataMigration_TEST.cls

```diff
@@ -131,6 +131,22 @@ private class RD2_DataMigration_TEST {
             'This RD should be in Enhanced RD format');
     }
 
+     /**
+     * @description Verifies that a schedule record is created for Enhanced Recurring Donation record.
+     */
+    @IsTest
+    private static void shouldCreateScheduleForEnhancedRDs() {
+        List<RecurringDonationSchedule__c> rdSchedules  = new List<RecurringDonationSchedule__c>();
```

Comment:
- The first argument in `System.assert` is expected value and the second one is actual value.
- The schedule record can be built only for the enhanced RD, so we need to convert the record before creating the schedule.
- Do we need to check any field on the schedule?

Based on the above, we can rewrite the test as:
```
Id contactId = UTIL_UnitTestData_TEST.mockId(Contact.SObjectType);
npe03__Recurring_Donation__c rd = getNewRecurringDonationEnhancedFormat(contactId);

RD2_DataMigrationMapper mapper = new RD2_DataMigrationMapper(rd);
mapper.convertToEnhancedRD();

List<RecurringDonationSchedule__c> schedules = mapper.getScheduleRecords();

System.assertEquals(1, schedules.size(), 'Number of schedules should match');
```